cmd/tether/attach.go

```diff
@@ -437,8 +446,15 @@ func (t *attachServerSSH) run() error {
 			log.Debugf("reader/writers bound for channel for %s", sessionid)
 
 			go t.channelMux(requests, session, detach)
-		}
 
+			// FIXME: This needs to be runblock instead of Attach
+			if session.Attach && session.Started != "true" {
+				// make sure that portlayer received the container id back
+				log.Debugf("Unblocking the launch of %s", sessionid)
+				session.ClearToLaunch <- <-t.askedAndAnswered
```

Comment:
I believe it does. t.askedAndAnswered will block until we close it and then we will send the zero value of the channel down to CleartToLaunc (reading from a closed channel sends its zero value)